install/install.sh

```
@@ -0,0 +1,521 @@
+# Main function
+main() {
+    print_header "Scopes Installer"
+    
+    # Check prerequisites
+    if ! command -v curl >/dev/null 2>&1 && ! command -v wget >/dev/null 2>&1; then
+        print_error "Either curl or wget is required for installation"
+        exit 1
+    fi
+    
+    # Detect platform and architecture
+    local platform=$(detect_platform)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)